src/utils/offchainState.ts

```diff
@@ -4,24 +4,31 @@ import path from 'path'
 import { createLogger } from '@subsquid/logger'
 import assert from 'assert'
 import { uniqueId } from './crypto'
+import { defaultNotificationPreferences } from './notification/helpers'
+import { NextEntityId } from '../model'
 
 const DEFAULT_EXPORT_PATH = path.resolve(__dirname, '../../db/export/export.json')
 
 const exportedStateMap = {
   VideoViewEvent: true,
   ChannelFollow: true,
   Report: true,
+  Exclusion: true,
+  ChannelVerification: true,
+  ChannelSuspension: true,
   GatewayConfig: true,
   NftFeaturingRequest: true,
   VideoHero: true,
   VideoFeaturedInCategory: true,
   EncryptionArtifacts: true,
   SessionEncryptionArtifacts: true,
+  OffChainNotification: true,
+  RuntimeNotification: ['mail_sent', 'in_app_read'],
```

Comment:
I can't find any entities by these names. Are you sure these Entities exist

Reply:
I have deleted these two entities and added the one for `Notification`